docs/integrations/glue_metastore.md

```diff
@@ -0,0 +1,394 @@
+              SerdeInfo:
+                SerializationLibrary: "org.apache.hadoop.hive.serde2.OpenCSVSerde"
+                Parameters:
+                  separatorChar: ","
+            Parameters: 
+              classification: "csv"
+              "skip.header.line.count": "1"
+```
+
+  </div>
+  <div markdown="1" id="single-hook">
+##### Spark Parquet Example
+
+The following snippet is a reference example of a table in parquet format input that can be used as `table_input`. 
+
+```yaml
+catalog:
+  table_input:
+    StorageDescriptor: 
+      InputFormat: "org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat"
+      OutputFormat: "org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat"
+      SerdeInfo:
+          SerializationLibrary: "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe"
+    Parameters: 
+      classification: "parquet"
+      EXTERNAL: "TRUE"
+      "parquet.compression": "SNAPPY"
+```
+
+  </div>
+  <div markdown="1" id="multiple-hooks">
+##### Multiple Hooks / Inline script
+
+The following example demonstrates how to separate the symlink and glue exporter into building blocks running in separate hooks.
+It also shows how to run the lua script inline instead of a file, depending on user preference.
+
+```yaml
+name: Animal Table Exporter
+on:
+  post-commit:
+    branches: ["main"]
+hooks:
+  - id: symlink_exporter
+    type: lua
+    properties:
+      args:
+        aws:
+          aws_access_key_id: "<AWS_ACCESS_KEY_ID>"
+          aws_secret_access_key: "<AWS_SECRET_ACCESS_KEY>"
+          aws_region: "<AWS_REGION>"
+        table_source: '_lakefs_tables/animals.yaml'
+      script: |
+        local exporter = require("lakefs/catalogexport/symlink_exporter")
+        local aws = require("aws")
+        local table_path = args.table_source
+        local s3 = aws.s3_client(args.aws.aws_access_key_id, args.aws.aws_secret_access_key, args.aws.aws_region)
+        exporter.export_s3(s3, table_path, action, {debug=true})
+  - id: glue_exporter
+    type: lua
+    properties:
+      args:
+        aws:
+          aws_access_key_id: "<AWS_ACCESS_KEY_ID>"
+          aws_secret_access_key: "<AWS_SECRET_ACCESS_KEY>"
+          aws_region: "<AWS_REGION>"
+        table_source: '_lakefs_tables/animals.yaml'
+        catalog:
+          db_name: "my-glue-db"
+          table_input: # add glue table input here 
+      script: |
+        local aws = require("aws")
+        local exporter = require("lakefs/catalogexport/glue_exporter")
+        local glue = aws.glue_client(args.aws.aws_access_key_id, args.aws.aws_secret_access_key, args.aws.aws_region)
+        exporter.export_glue(glue, args.catalog.db_name, args.table_source, args.catalog.table_input, action, {debug=true})  
+```
+
+  </div>
+</div>
+
+Once we are adding the script and the action file we can commit it and since the action is configured to `post-commit` it will run after the commit! 
```

Comment:
```suggestion
Adding the script and the action file to the repository and commit it.  This is a post-commit action - it will run after the commit! 
```